# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/AbortSignal.webidl
# Commit: d27c32ae0908
# Full Hash: d27c32ae0908c0b5d2c9213b9a624e3d0b71e2d5
# Author: Vincent Hilla <vhilla@mozilla.com>
# Date: 2024-01-23 21:57:50
# Regressor Bug: 1830781
# File Overlap Count: 1
# Description:
#   Bug 1830781 - Implement AbortSignal.any() r=saschanaz,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D198011
# ==============================================================================

diff -r 8af7503ecb57 -r d27c32ae0908 dom/webidl/AbortSignal.webidl
--- a/dom/webidl/AbortSignal.webidl	Tue Jan 23 08:04:52 2024 +0000
+++ b/dom/webidl/AbortSignal.webidl	Tue Jan 23 08:43:07 2024 +0000
@@ -12,6 +12,7 @@
   [NewObject] static AbortSignal abort(optional any reason);
   [Exposed=(Window,Worker), NewObject, Throws]
   static AbortSignal timeout([EnforceRange] unsigned long long milliseconds);
+  [NewObject] static AbortSignal _any(sequence<AbortSignal> signals);
 
   readonly attribute boolean aborted;
   readonly attribute any reason;